templates/base.html

```diff
@@ -38,5 +38,7 @@
 
 {% include "security/_messages.html" %}
 {% block body %}{% endblock %}
-
+    
+<main>
```

Comment:
Taking the opening tag also mean you should take the closing tag along.

Reply:
👍